config.yml

```diff
@@ -39,11 +39,10 @@ observer:
 postgres:
   uri: postgresql://user:pass@localhost/my_db?sslmode=disable
 
-# [BNB] Binance DEX: https://wallet.binance.org
-#       Binance Chain: https://explorer.binance.org
+# [BNB] Binance DEX: https://www.binance.org/
 binance:
-  api: https://explorer.binance.org/api/v1
-  dex: https://dex.binance.org/api
+  api: https://dex.binance.org/api
+  dex: https://explorer.binance.org/api/v1
```

Comment:
Move the 'v1' version designator from config to client code -- versioning is not a configuration issue, but a code issue.  If version is changed, likely code has to be changed as well.
Remove v1 from config, and add it to platform/binance/dex_client.go .